cpp/tests/strings/replace_tests.cpp

```diff
@@ -337,30 +337,6 @@ TEST_F(StringsReplaceTest, ReplaceMulti)
   }
 }
 
-TEST_F(StringsReplaceTest, ReplaceNulls)
-{
-  std::vector<const char*> h_strings{"Héllo", "thesé", nullptr, "ARE THE", "tést strings", ""};
-
-  cudf::test::strings_column_wrapper strings(
-    h_strings.begin(),
-    h_strings.end(),
-    thrust::make_transform_iterator(h_strings.begin(), [](auto str) { return str != nullptr; }));
-  auto strings_view = cudf::strings_column_view(strings);
-
-  {
-    auto results = cudf::strings::replace_nulls(strings_view, cudf::string_scalar("___"));
-    std::vector<const char*> h_expected{"Héllo", "thesé", "___", "ARE THE", "tést strings", ""};
-    cudf::test::strings_column_wrapper expected(h_expected.begin(), h_expected.end());
-    CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected);
-  }
-  {
-    auto results = cudf::strings::replace_nulls(strings_view);
-    std::vector<const char*> h_expected{"Héllo", "thesé", "", "ARE THE", "tést strings", ""};
-    cudf::test::strings_column_wrapper expected(h_expected.begin(), h_expected.end());
-    CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected);
-  }
-}
-
```

Comment:
Should this test be moved to the base replace nulls or is it covering strings sufficiently now?

Reply:
The base one seems to have strings well covered.
https://github.com/rapidsai/cudf/blob/d501d2c0b9cc7309f08646a2677c019e6f7906b6/cpp/tests/replace/replace_nulls_tests.cpp#L75-L238